Made exceptions caught by get_component_class more specific (#4166)
RasaHQ/rasa: CHANGELOG.rst

- ``RasaChatInput`` fetches the public key from the Rasa X API. The key is used to
  decode the bearer token containing the conversation ID. This requires
  ``rasa-x>=0.20.2``.

> - more specific exception message from ``get_component_class()`` depending on whether component's path or

Reviewer: ```suggestion
- more specific exception message when loading custom components depending on whether component's path or
```
